src/javascript/static/pages/responsible_trading.js

```diff
@@ -0,0 +1,21 @@
+const Client = require('./../../app/base/client');
+const getElementById = require('../../../javascript/_common/common_functions').getElementById;
+const LocalStore = require('../../_common/storage').LocalStore;
+
+const ResponsibleTrading = (() => {
+
+    const onLoad = () => {
+        const residence = Client.get('residence');
+        const current_loginid = LocalStore.get('active_loginid');
```

Comment:
Should we use `ClientBase` instead to get the `loginid`?

Reply:
In my new commit, I used `landing company` instead of `loginid`